actions-on-google.js

```diff
@@ -81,9 +85,21 @@ class ActionsOnGoogle {
     }
 
     setLocale(locale) {
+        if (!SUPPORTED_LOCALES.concat(Object.keys(FALLBACK_LOCALES)).includes(locale)) {
+            throw new Error(`Unsupported locale: ${locale}`);
```

Comment:
If we introduce a new language and the library is not updated immediately, or the developer doesn't update the language, changing locales would fail. However, the failure would only happen in the abstracted layer. The underlying API call would still succeed.

Instead of throwing an error, should we instead log a warning and still allow them to set the locale? While they'd be unable to use the descriptive API, they could still use the `.send` method with the text until we create a new patch with the new locale.

Reply:
@Fleker Thank you for the advice! I agree all that. Actually, by using `.send` only, developers can do test-cases with any language, but I didn't notice that, sorry. I will write a code to output a warning log, instead of the exception throwing.